.github/workflows/test.yml

```diff
@@ -1,21 +1,20 @@
-name: lint_python
```

Comment:
Please undo this name change.

Reply:
why? this workflow does more than just linting

Reply:
It is more linting (bandit, codespell, flake8, mypy, safety) then testing (pytest) or reformatting (black, isort, pyupgrade).  `lint_python` is an Action that I broadly adopt to many projects so it is better to keep the naming unless we make substantial changes in the tools used.